internal/scorecard/alpha/testpod.go

```diff
@@ -89,6 +122,7 @@ func getPodLog(ctx context.Context, client kubernetes.Interface, pod *v1.Pod) (l
 	return buf.Bytes(), err
 }
 
+// deletePods deletes a collection of pods that match a predefined selector value
 func (o Scorecard) deletePods(ctx context.Context) error {
```

Comment:
Make this a method on `PodTestRunner`?

Reply:
changed deletePods to be a method on PodTestRunner.